packages/@tailwindcss-upgrade/src/index.ts

```diff
@@ -124,6 +123,9 @@ async function run() {
       let config = await prepareConfig(sheet.linkedConfigPath, { base })
       configBySheet.set(sheet, config)
 
+      info(
+        `Migrating JavaScript configuration file: ${highlight(relative(config.configFilePath, base))}`,
+      )
```

Comment:
We could be dealing with multiple config files, therefore this log has been moved inside of the loop.

Slightly updated the message and included the linked config file.